### DISCUSSION TITLE
Is there a way to get the auth code through console?

### DISCUSSION THREAD

**Discussion by radtixx** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#661] [Q&A]
Hi guys. I'm using a Raspbian Lite without "x" as a server.
My question is, would be possible to login anyway to twitch without desktop?
Tried through w3m, lynx, elinks console browsers but it doesn't work. I need the verification code in order to use miner, and I won't to install PIXEL or any different desktop.
Thanks in advance!

**Comment by rdavydov** [comment]
Hi. Yes, it is currently possible with my fork https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2/
